com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ArchiveBuilder.java

```diff
@@ -302,7 +294,9 @@ public void write(RandomAccessFile archiveIndex, RandomAccessFile archiveData, L
         // create the executor service to write entries in parallel
         int nThreads = project.getMaxCpuThreads();
         logger.info("Creating archive entries with a fixed thread pool executor using %d threads", nThreads);
-        this.executorService = Executors.newFixedThreadPool(nThreads);
+        ExecutorService multiThreadedExecutorService = Executors.newFixedThreadPool(nThreads);
+        // temp workaround for a bug in the ZipPublisher
+        ExecutorService singleThreadedExecutorService = Executors.newFixedThreadPool(1);
```

Comment:
Actual workaround. Single threaded archive creation works good, let's do it this way for now until we fix it as part of  https://github.com/defold/defold/issues/10012